builder/jdcloud/credential_config.go

```diff
@@ -67,19 +67,19 @@ func (jd *JDCloudCredentialConfig) ValidateKeyPair() error {
 	return nil
 }
 
-func (config *JDCloudCredentialConfig) validateRegion() error {
```

Comment:
In other `JDCloudCredentialConfig` methods the method receiver is called `jd` and I think it would be best to keep it named the same way throughout the code. It would be more idiomatic.
The docs can be found here: https://talks.golang.org/2014/names.slide#11

Reply:
i renamed all receivers to ```jd```

Reply:
Oops, yes !